PowerDocu.FlowDocumenter/FlowDocumentationContent.cs

```diff
@@ -19,7 +19,7 @@ class FlowDocumentationContent
         public FlowDocumentationContent(FlowEntity flow, string path, FlowActionSortOrder sortOrder = FlowActionSortOrder.SortByName)
         {
             NotificationHelper.SendNotification("Preparing documentation content for " + flow.Name);
-            folderPath = path + CharsetHelper.GetSafeName(@"\FlowDoc - " + flow.Name + @"\");
+            folderPath = path + CharsetHelper.GetSafeName(@"\FlowDoc " + flow.Name + @"\");
```

Comment:
The '-' is read in DevOps Wiki's as a space, which got the wiki structure very confused so this had to be removed